docs/installation.rst

```diff
@@ -84,7 +84,7 @@ You may already have a recent version of Dask_ installed, in which case you can
 Building Modin from Source
 --------------------------
 
-If you're planning on contributing_ to Modin, you will need to ensure that you are
+If you're planning on :doc:`contributing </contributing>` to Modin, you will need to ensure that you are
```

Comment:
If a doc is located in the same folder, do we need to specify `/`?

Reply:
Yes, in our case we need to specify all files by template `</path>` where `/ ` is `docs/` folder.